handbook/sprint-process.md

```diff
@@ -1,35 +1,60 @@
+  - Punctuality, participation, transparency and open communication.
 
-#### Sprint Kickoff
+#### Sprint Planning
 
-- Cadence: Biweekly, on sprint Tuesdays (during the Data Leads meeting)
-  
+- Cadence: Biweekly, on the Tuesday of the week of the Sprint start, 1 hour timebox
+- Agenda: 
+  - Review current sprint and potential carryover into the next sprint.
+  - Agile Practitioner provides an estimated sprint capacity for the upcoming sprint.
+  - Agile Practioner & Data leads distribute a curated Sprint Planning Log for the upcoming sprint to the team for review and consideration.
+  - Discuss the value of the identified items and their alignment with project goals.
+  - Move items into the next sprint, review capacity, and make necessary adjustments.
+    
+- Expectations:
+  - By the end of the Sprint Planning session, issues for the next sprint will be identified, carryover items acknowledged, and the next sprint is in good shape and ready to be launched the next day. 
+
+#### Sprint Launch
+
+- Cadence: bi-weekly on sprint Wednesdays, 2:00pm ET, 15 minute timebox
 - Agenda: 
-  - Given that the upcoming sprint is already pre-populated, the data team leadership accounts for any work that has spilled over from the previous sprint, removing points as necessary to keep the total story points within our team's sprint capacity. 
-  - Once this process is complete, the Agile coach starts the new sprint. 
+  - The Agile Practitioner closes the current sprint.
+  - Any carryover issues from the previous sprint are moved into the next sprint or into the backlog.
+  - The Agile Practitioner starts the next sprint.
+  - Team members begin working on tasks and initiatives for the new sprint.
+
+ Expectations:
+  - The Agile Practitioner will close the current sprint, ensuring that all tasks and deliverables are completed or handled appropriately.
 
 
 #### Backlog Refinement
 
-- Cadence: Biweekly, on off-sprint Tuesdays
+- Cadence: Biweekly, on off-sprint Wednesdays, 1 hour timebox
   
-- Agenda: The team reviews any unpointed, unblocked tickets. After discussing the ticket, each team member provides a story point estimate for the ticket. When a consensus is reached, that story point value is added to the ticket.
+- Agenda:
+   - Review tickets in status DI Refinement.
+   - Address questions or concerns regarding deliverables, dependencies, and blockers and clarify goals, acceptance criteria, and review/testing requirements.
+   - Populate the Epic with child issues, providing high-level summaries and establishing linkages.
+   - Move issues to a ready state ("TO DO") for assignment.
 
 - Expectations:
-  - Team members review tickets and add questions and comments to the tickets in advance of refinement to facilitate better team discussions. 
+  - Team changes status of  any tickets in need of refinement to: DI Refinement.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question/thought:** ‏
>- Team changes status of any tickets in need of refinement to: DI Refinement.

Would the team be identifying these tickets during refinement? Or beforehand? I wonder if new DI tickets could have "DI Refinement" as their initial status to avoid this manual step.

Reply:
This step should be done prior to refinement, I will update the language.  I am not having any luck with creating an automation to change the status of unpointed new tickets to DI Refinement, so I will work with Ben in Q1 on other possible approaches.